WalletWasabi.Fluent/ViewModels/Wallets/Buy/BuyViewModel.cs

```diff
@@ -189,7 +189,7 @@ private async Task CreateAndAddOrdersAsync(IReadOnlyList<Conversation> conversat
 
 	private Country? GetCountryFromConversation(Conversation conversation)
 	{
-		var countryName = conversation.ChatMessages.FirstOrDefault(x => x.MetaData?.Tag == ChatMessageMetaData.ChatMessageTag.Country)?.Message;
```

Comment:
I added this because it was erroring out, seems there are several places where metadata is initialized to null? Maybe a leftover from an older version of the code.

Reply:
It was a mistake, it should never be null. We fixed the issue meantime.